sensors/listener.go

```diff
@@ -220,6 +220,21 @@ func (sensorCtx *SensorContext) listenEvents(ctx context.Context) error {
 				})
 				if err != nil {
 					triggerLogger.Warnf("failed to trigger actions, %v", err)
+					if trigger.DlqTrigger != nil {
+						dlqRetryStrategy := trigger.DlqTrigger.RetryStrategy
+						if dlqRetryStrategy == nil {
+							dlqRetryStrategy = &apicommon.Backoff{Steps: 1}
+						}
+
+						triggerLogger.Warnf("invoking dlqTrigger, %v", err)
```

Comment:
This feels like `debug` level logging  instead of warning. Also it feels like `err` is double logged (as it is already logged on line 222).

Reply:
done!